TranslatorReasonerAPI.yaml

```diff
@@ -295,18 +295,26 @@ components:
     KnowledgeGraph:
       type: object
       description: >-
-        A thought graph associated with this result. This will commonly be a
-        linear path subgraph from one concept to another, but related items
-        aside of the path may be included.
+        A thought graph associated with this result. This will commonly be
+        a linear path subgraph from one concept to another, but related items
```

Comment:
My impression is that for most questions, the knowledge graph is not linear, even when the query graph is. For one-hop questions, for instance, it's a star, with each Result corresponding to one point of the star.

Reply:
Sure. You are more in the know about  this.  I'll modify this  accordingly  and  re-commit.